examples/web-nextjs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "web-nextjs",
+  "description": "NextJS example setup with Reassure",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@types/node": "18.14.1",
+    "@types/react": "18.0.28",
+    "@types/react-dom": "18.0.11",
+    "eslint": "8.34.0",
+    "eslint-config-next": "13.2.1",
+    "next": "13.2.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "typescript": "4.9.5"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.0",
+    "@testing-library/react": "^14.0.0",
+    "@types/jest": "^29.4.0",
+    "jest": "^29.4.3",
+    "jest-environment-jsdom": "^29.4.3",
+    "reassure": "^0.7.1"
```

Comment:
Did you install `reassure` using `yarn add reassure` inside the example folder? I just want to double-check because bob is handling building libraries and linking them between packages out-of-the-box so we definitely **do not** want to do it any other way because you will end up working with a version of reassure not aligned within the monorepo. 

Please confirm how you went about it @kbieganowski ;)